src/finch/tensor.py

```diff
@@ -897,8 +949,32 @@ def atanh(x: Tensor, /) -> Tensor:
     return x._elemwise_op("atanh")
 
 
-def atan2(x: Tensor, other: Tensor, /) -> Tensor:
-    return x._elemwise_op("atand", other)
+def atan2(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1._elemwise_op("atand", x2)
+
+
+def trunc(x: Tensor, /) -> Tensor:
+    return x._elemwise_op("trunc")
+
+
+def square(x: Tensor, /) -> Tensor:
+    return x ** Tensor(2)
+
+
+def logaddexp(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return log(exp(x1) + exp(x2))
+
+
+def logical_and(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1._elemwise_op("Finch.and", x2)
+
+
+def logical_or(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1._elemwise_op("Finch.or", x2)
+
+
+def logical_xor(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1._elemwise_op("Finch.xor", x2)
```

Comment:
Can we add tests for these?

Reply:
They are all tested. Please see `test_elemwise_ops_2_args` and `test_elemwise_ops_1_arg` changes.